doc/api/cluster.md

```diff
@@ -856,6 +856,8 @@ socket.on('data', (id) => {
 });
 ```
 
+[`.fork()`]: #cluster_cluster_fork_env
```

Comment:
```suggestion
[`.fork()`]: #cluster_cluster_fork
```

Reply:
@trivikr I don’t understand…? https://nodejs.org/api/cluster.html#cluster_cluster_fork_env *is* the link to the method’s definition.

Reply:
My bad, it was not linking to the correct method definition when I'd checked while reviewing. It must have been a miss from my side.

Confirmed that https://nodejs.org/api/cluster.html#cluster_cluster_fork_env links to the `cluster.fork([env])`, resolving this comment